src/components/Pools/AddLiquidity/ConfirmPoolDrawer/index.tsx

```diff
@@ -234,7 +234,7 @@ const ConfirmSwapDrawer: React.FC<Props> = (props) => {
 
           <Stat
             title={`PGL`}
-            stat={noLiquidity ? '-' : `${numeral(liquidityMinted?.toSignificant(6)).format('0.00a')}`}
+            stat={noLiquidity ? '-' : `${numeral(liquidityMinted?.toFixed(4)).format('0.00a')}`}
```

Comment:
getting this error on add liquidity confirm drawer in hedera chain

![Screenshot 2023-04-06 at 11 04 33 PM](https://user-images.githubusercontent.com/6604146/230453821-abf749cb-9fc8-439b-8c7e-c025c20d2520.png)